WireAPI/Sources/WireAPI/APIs/AuthenticationAPI/AuthenticationAPI.swift

```diff
@@ -20,6 +20,10 @@ import Foundation
 
 public protocol AuthenticationAPI {
 
+    /// Get on-prem config `URL` for domain
+
+    func getOnPremConfigURL(forDomain domain: String) async throws -> DomainInfo
```

Comment:
@johnxnguyen Not sure to have this endpoint here or move it to the `BackendInfoAPI`

Reply:
For me it's fine to have it here, it's more relevant to authentication